### DISCUSSION TITLE
How to make use of the EasingFunction?

### DISCUSSION THREAD

**Discussion by CJ13th** [nut-tree/nut.js#424] [Q&A] [answered]
Hi, can anyone please give quick a code snippet example of how to make use of the easing functions that are mentioned in the docs? I've been trying to utilise them but I'm struggling to get it work.

Any help much appreciated!

**Comment by s1hofmann** [comment, marked as answer]
There's now a sample at https://github.com/nut-tree/trailmix/tree/master/packages/mouse-test#easingfunction, official docs will be updated soon.
